cmd/otelcol/config/collector/fargate_config.yaml

```diff
@@ -52,6 +52,8 @@ receivers:
     listenAddress: 0.0.0.0:9080
   smartagent/ecs-metadata:
     type: ecs-metadata
+    metadataEndpoint: "${ECS_TASK_METADATA_ENDPOINT}"
+    statsEndpoint: "${ECS_TASK_STATS_ENDPOINT}"
```

Comment:
Can you please move fargate related changes to a different PR?

Reply:
Yes